src/app/core/services/youtube.search.spec.ts

```diff
@@ -47,7 +47,7 @@ describe('Youtube Search Service', () => {
   });
 
   it('should search with same value when searching more and trim', () => {
-    const query = 'ozrics';
+    let query = 'ozrics';
```

Comment:
It's not clear why the `const` keyword was changed to `let` for the `query` variable. If the value of `query` is not being reassigned in this block of code, it would be better to keep it as a `const` to prevent accidental reassignment and to make the code more readable.